src/components/admin/klassen/KlasseDelete.vue

```diff
@@ -22,59 +22,68 @@
     (event: 'onClose'): void;
   };
 
+  enum State {
+    CONFIRM,
+    LOADING,
+    RESULT,
+  }
```

Comment:
```suggestion
  enum State {
    LOADING,
    CONFIRM,
    SUCCESS,
  }
```
I find the word "SUCCESS" more meaningful here because it's tied to the success message, what do you think?

Also is "LOADING" explicitly used or is it just a fall back?